.github/workflows/E2ETest.yml

```diff
@@ -14,8 +14,10 @@ on:
       - 'test/e2e/**'
 
 jobs:
-  build:
+  e2e-azurestorage:
```

Comment:
Not sure if I understand it right --based on the workflow title, this e2e tests should be for .NET isolated apps, not for .NET in-proc apps. But why we only build the pkg webjobs.extensions.durabletask, and remove worker.extensions.durabletask ?

Reply:
See comment on build-e2e-test.ps1 - the Worker extension gets built as a project reference, but the WebJobs extension will fall back on possibly invalid cached versions in the local NuGet cache unless we explicitly build it in this way